### PR TITLE
Language read refactor

### PR DESCRIPTION
**Pull request by mikecoll**
Language Readone Refactor

### CODE REVIEW

**Review by CarsonF — COMMENTED** [review]
Lots of commented out code here. That only adds more clutter. git should be used for previous versions of code.

**Comment by izquierdoalfredo** [comment]
It's not finished yet, I will clean code after finish.

**Comment by CarsonF** [comment]
Ok got it. If it's not ready for review please open the PR as a draft.

**Comment by izquierdoalfredo** [comment]
@CarsonF Can you please check this PR?
It's ready for review.

File: src/components/language/language.service.ts [hunk 1]
```diff
@@ -388,65 +381,119 @@ export class LanguageService {
...
       .call(matchRequestingUser, session)
-      .call(matchUserPermissions, 'Language', langId)
-      .call(addAllSecureProperties, ...props)
-      .call(addAllMetaPropertiesOfChildBaseNodes, ...childBaseNodeMetaProps)
-      .with([
-        ...props.map(addPropertyCoalesceWithClause),
-        ...childBaseNodeMetaProps.map(addShapeForChildBaseNodeMetaProperty),
-        ...baseNodeMetaProps.map(addShapeForBaseNodeMetaProperty),
-        'node',
+      .match([node('node', 'Language', { active: true, id: langId })])
```
Comment (frankyma) [inline src/components/language/language.service.ts:398]: Nitpick: give a more specific name for 'node', maybe 'languageNod'.  

I'm just reading this PR to understand the new query changes but betting labeling would help a beginner like me follow along

Comment (CarsonF) [reply src/components/language/language.service.ts:398]: I believe `node` almost always represents the thing you are fetching. We are in the `LanguageService` so it represents a language.

Comment (michaelmarshall) [reply src/components/language/language.service.ts:398]: its convention used in the `query.helper.ts` file. It must be `node`.

Comment (frankyma) [reply src/components/language/language.service.ts:398]: got it, thanks guys

**Pull request closed without merging**